tools/update_tool

```diff
@@ -0,0 +1,439 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import sys
+import tempfile
+import shlex
+import subprocess
+from abc import ABC, abstractmethod
+import columnify
+
+RESET = "\033[0m"
+GREEN = "\033[32m"
+BOLD = "\033[1m"
+RED = "\033[31m"
+YELLOW = "\033[33m"
+STRIKE = "\033[9m"
+
+
+class Report(ABC):
+
+    def __init__(self):
+        self.header()
+
+    @abstractmethod
+    def header(self):
+        pass
+
+    @abstractmethod
+    def row(self, name, mstatus, istatus):
+        pass
+
+
+class CliReport(Report):
+
+    def __init__(self):
+        self.terminal_width = os.get_terminal_size().columns
+        section1 = 0.7
+        section2 = 0.15
+        section3 = 0.15
+        self.nwidth = int(self.terminal_width*section1)-5
+        self.mwidth = int(self.terminal_width*section2)-2
+        self.iwidth = int(self.terminal_width*section3)-1
+        super().__init__()
+
+    @classmethod
+    def truncate(cls, s, l):
+        if len(s) > l:
+            s = f"{s[0:l-3]}..."
+        return s
+
+    def adjust(self, s, l):
+        return self.truncate(s, l).ljust(l)
+
+    def header(self):
+        print("| ", "DB entry".ljust(self.nwidth), end="| ")
+        print(self.adjust("manifest", self.mwidth), end="| ")
+        print(self.adjust("image-info", self.iwidth), end="|\n")
+
+        print(f"|-{'-'*(self.nwidth+1)}", end="|-")
+        print(f"{'-'*self.mwidth}", end="|-")
+        print(f"{'-'*self.iwidth}", end="|\n")
+
+    status_map = {"added": f"{BOLD}{GREEN}",
```

Comment:
Very minor but I like having my class attributes near the top of the class (listed first).